_posts/2021-07-13-odh-release-1.1.0-blog.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: blog
+author: nakfour
+title:  Open Data Hub 1.1.0 Release Guide
+preview: Open Data Hub 1.1.0 was just released
+date: 2021-07-13
+categories: features, road map, release
+---
+
+What is new?
+------
+In Open Data Hub version 1.1.0 the community focused on hardening the JupyterHub deployment, providing new and improved JupyterHub spawner UI, integrating Open Data Hub dashboard with Openshift oauth, adding [Kubeflow 1.3 Openshift distribution stack](https://github.com/kubeflow/manifests/tree/v1.3.0/distributions/stacks/openshift) along with adding new components such as Trino and Openshift Pipelines.  ODH version 1.1.0 also comes with an Operator Level 4 verification indicating "Deep Insight" status after enabling more monitoring and logging. 
+ 
+An important note to users, ODH 1.1.0 includes two example KFDefs for installation on OpenShift 4.3 (or later) :
+1. ODH components [KFDef](https://raw.githubusercontent.com/opendatahub-io/odh-manifests/master/kfdef/kfctl_openshift.yaml)
```

Comment:
```
 repos:
  - name: kf-manifests
    uri: https://github.com/kubeflow/manifests/archive/v1.2-branch.tar.gz
  - name: manifests
    uri: https://github.com/opendatahub-io/odh-manifests/tarball/master
```

1. Does the KF manifest needs to changes to `1.3-branch`?
2. Does the ODH manifest need to be on the `v1.1.0` tag and not `master`?

Reply:
Good catch, thanks fixed it